src/App.tsx

```diff
@@ -1,14 +1,11 @@
+import { RouterProvider } from 'react-router-dom';
 import { router } from './router/router';
 import './styles/_main.scss';
-import { RouterProvider } from 'react-router-dom';
-
-function App() {
 
+export function App() {
   return (
     <div className="App">
       <RouterProvider router={router} />
     </div>
   );
 }
-
-export default App;
```

Comment:
Why did you do this change?

Reply:
Because named export is more strict and forbid to import some component with the other name that was given to it